cv32e40s/tests/programs/custom/csr_priv_gen_test/README.md

```diff
@@ -1,5 +1,9 @@
-# Python generator files 
+Contains generated tests on U-mode csr instructions to insert illegall access
```

Comment:
I was looking for an explanation of what the test does, and when I found it I just copy-pasted it into the readme so it is easier to find.